internal/compiler/passes.rs

```diff
@@ -56,6 +57,16 @@ use crate::expression_tree::Expression;
 use crate::namedreference::NamedReference;
 use smol_str::SmolStr;
 
+pub fn ignore_debug_hooks(expr: &Expression) -> &Expression {
```

Comment:
Not sure if this belong in this module.

Maybe in `impl Expression` ?

Reply:
This will be moved in the next push.